apps/blockchain/contracts/starknet/src/bridge/escrow_bridge.cairo

```diff
@@ -0,0 +1,70 @@
+#[abi]
+trait IBridgeEscrow {
+    fn lock_tokens();
+    fn unlock_tokens();
+}
+
+#[contract]
+mod EscrowBridge {
+    use starknet::ContractAddress;
+    use starklane_contracts::interfaces::IERC721::IERC721Dispatcher;
+    use starklane_contracts::interfaces::IERC721::IERC721DispatcherTrait;
+    use starklane_contracts::utils::serde;
+    use array::SpanTrait;
+    use array::ArrayTrait;
+    use starknet::get_contract_address;
+    use starknet::get_caller_address;
+    use starknet::contract_address_const;
+
+    struct Storage {
+        bridge_address: ContractAddress,
+        owner: ContractAddress,
+        depositors: LegacyMap::<(ContractAddress, u256), ContractAddress>,
+    }
+
+    #[constructor]
+    fn constructor(_owner: ContractAddress, _bridge_address: ContractAddress) {
```

Comment:
We should take the overall conventions, which seems to be using underscore in front of all storage variables, or adding a prefix.
This will add more consistency in our overall code base. Not sure yet what's the best way to go, let's discuss it! :)

Reply:
Yes sure, nice idea. What would be the best place to create a document like this? Notion or GitHub Discussion? Or simply in a .md file on the repo?

FYI I'm not yet familiar with the best practices or standards in Cairo in terms of naming. If you have any ideas, don't hesitate to let me know @glihm @kwiss